lib/AST/ASTDemangler.cpp

```diff
@@ -479,9 +479,9 @@ Type ASTBuilder::createImplFunctionType(
     break;
   }
 
-  auto einfo = SILFunctionType::ExtInfo(representation,
-                                        flags.isPseudogeneric(),
-                                        !flags.isEscaping());
+  auto einfo = SILFunctionType::ExtInfo(
```

Comment:
Is there a mangling for these flags?  If so, it seems like it should be plumbed in here for the demangler to use.

Reply:
We've got some mangling in the `tensorflow` branch. I will upstream it in a separate PR. I'll add a TODO comment here referencing the ticket: https://bugs.swift.org/browse/TF-948